src/main/java/cc/polyfrost/oneconfig/gui/elements/config/ConfigDropdown.java

```diff
@@ -199,7 +202,7 @@ public void drawLast(long vg, int x, int y, InputHandler inputHandler) {
                 }
                 if (optionHovered && inputHandler.isClicked(!dragging)) {
                     try {
-                        set(ArraysKt.indexOf(this.options, option));
+                        set(ArraysKt.indexOf(originalOptions, option));
```

Comment:
I'd suggest not using ArraysKt or any kotlin runtime classes in Java code, which is why I rewrote the original `ArraysKt.filter` to use a for-loop, but in this instance it might be more troublesome than anything.

Reply:
its just a nice util thing and I cba